chain-spec-generator/Cargo.toml

```diff
@@ -12,23 +12,9 @@ serde_json = { workspace = true }
 serde = { features = ["derive"], workspace = true }
 
 polkadot-runtime = { workspace = true }
```

Comment:
We should not need the `chain-spec-generator` anymore with your changes? We can just use the `chain-spec-generator` from Substrate.

Reply:
yes, that is the plan, I created a separate issue for that: https://github.com/polkadot-fellows/runtimes/issues/395